packages/react-components/react-datepicker-compat/src/index.ts

```diff
@@ -1,41 +1,11 @@
-export { AnimationDirection, Calendar, calendarClassNames, useCalendarStyles_unstable } from './Calendar';
-export type { CalendarNavigationIcons, CalendarProps, CalendarStyleProps, CalendarStyles, ICalendar } from './Calendar';
+export { AnimationDirection, calendarClassNames } from './Calendar';
+export type { CalendarProps, ICalendar } from './Calendar';
 
-export { CalendarDay, calendarDayClassNames, useCalendarDayStyles_unstable } from './CalendarDay';
-export type { CalendarDayProps, CalendarDayStyleProps, CalendarDayStyles, ICalendarDay } from './CalendarDay';
+export { calendarDayClassNames } from './CalendarDay';
+export type { CalendarDayProps, ICalendarDay } from './CalendarDay';
 
-export {
-  CalendarDayGrid,
-  calendarDayGridClassNames,
-  extraCalendarDayGridClassNames,
-  useCalendarDayGridStyles_unstable,
-} from './CalendarDayGrid';
-export type {
-  CalendarDayGridProps,
-  CalendarDayGridStyleProps,
-  CalendarDayGridStyles,
-  DayInfo,
-  ICalendarDayGrid,
-  WeekCorners,
-} from './CalendarDayGrid';
-
-export { CalendarMonth, calendarMonthClassNames, useCalendarMonthStyles_unstable } from './CalendarMonth';
-export type { CalendarMonthProps, CalendarMonthStyleProps, CalendarMonthStyles, ICalendarMonth } from './CalendarMonth';
-
-export { calendarPickerClassNames, useCalendarPickerStyles_unstable } from './CalendarPicker';
-export type { CalendarPickerStyleProps, CalendarPickerStyles } from './CalendarPicker';
-
-export { CalendarYear, useCalendarYearStyles_unstable } from './CalendarYear';
-export type {
-  CalendarYearHeaderProps,
-  CalendarYearProps,
-  CalendarYearRange,
-  CalendarYearRangeToString,
-  CalendarYearStrings,
-  CalendarYearStyleProps,
-  CalendarYearStyles,
-  ICalendarYear,
-} from './CalendarYear';
+export { calendarMonthClassNames } from './CalendarMonth';
+export type { CalendarMonthProps, ICalendarMonth } from './CalendarMonth';
 
 export {
   DatePicker,
```

Comment:
The utils imports are a little ambiguous if they should be removed. We use some of them in our examples and it makes sense to provide them for the user. Would it be fine to keep the date calculation related exports from below?

Reply:
I think it makes sense, especially given people can use them to define things like bounds, filters, etc.